admin/client/utils/queryParams.js

```diff
@@ -45,3 +45,11 @@ export function stringifyColumns (columns, defaultColumnPaths) {
 	if (columnString === defaultColumnPaths) columnString = undefined;
 	return columnString;
 }
+
+export function parametizeFilters (filterArray) {
```

Comment:
Sounds like a plan. Would you like a hand? Or are you comfortable with the changes that would need to be made there?